public_html/wp-content/themes/wporg-events-2023/patterns/home-page-callout.php

```diff
@@ -0,0 +1,30 @@
+
+<?php
+/**
+ * Title: Front Page
+ * Slug: wporg-events-2023/home-page-callout
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"padding":{"top":"5rem","right":"var:preset|spacing|edge-space","bottom":"5rem","left":"var:preset|spacing|edge-space"}}},"backgroundColor":"blueberry-1","textColor":"white","layout":{"inherit":true,"type":"constrained"}} -->
+<div class="wp-block-group alignfull has-white-color has-blueberry-1-background-color has-text-color has-background has-link-color" id="get-started" style="padding-top:5rem;padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:5rem;padding-left:var(--wp--preset--spacing--edge-space)"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"60px"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":"0px"}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"70%"} -->
+<div class="wp-block-column" style="flex-basis:70%"><!-- wp:paragraph {"style":{"layout":{"selfStretch":"fixed","flexSize":"70%"},"elements":{"link":{"color":{"text":"var:preset|color|white"}}}},"textColor":"white","fontSize":"heading-4","fontFamily":"eb-garamond"} -->
+<p class="has-white-color has-text-color has-link-color has-eb-garamond-font-family has-heading-4-font-size">If you can't find a WordPress event around your area, you can become an organizer
```

Comment:
I created the footer part because the mockup shows this on every page. Do you think there's a reason to limit it to the homepage?

Reply:
Not, this isn't finished so I haven't yet scanned all the pages and landed everything logically, but it does have to be in a pattern, `.php` for translation purposes.

Reply:
Good call, I added it back. 👍